webpack/webpack.ghpages.cjs

```diff
@@ -0,0 +1,21 @@
+'use strict';
+
+const { merge } = require('webpack-merge');
+
+const generateConfig = require('./webpack.config.cjs');
+const constants = require('./constants.cjs');
+
+// Add timerbar_test.html
+constants.cactbotModules['timerbarTest'] = 'ui/test/timerbar_test';
+constants.cactbotHtmlChunksMap['ui/test/timerbar_test.html'] = {
+  chunks: [
+    constants.cactbotModules.timerbarTest,
+  ],
+};
+
```

Comment:
This is a little bit unfortunate to repeat, especially if we add more things like this in the future.  I think I'd be fine not shipping timerbar_test to gh-pages.

Reply:
Okay